build.sbt

```diff
@@ -4,7 +4,10 @@ name := "toolform"
 
 organization := "au.com.agiledigital"
 
-scalaVersion := "2.12.4"
+//crossScalaVersions := Seq("2.13.1", "2.12.4")
```

Comment:
Do not commit dead code. Remove this commented code.

Reply:
Sorry Dan, I will do a clean up and recommit